src/Compilers/CSharp/Portable/Binder/Binder_Crefs.cs

```diff
@@ -737,7 +737,7 @@ private static ImmutableArray<Symbol> PerformCrefOverloadResolution(ArrayBuilder
                                 callingConvention: candidateMethodIsVararg ? Microsoft.Cci.CallingConvention.ExtraArguments : Microsoft.Cci.CallingConvention.HasThis,
                                 // These are ignored by this specific MemberSignatureComparer.
                                 containingType: null,
-                                name: null,
+                                name: string.Empty,
```

Comment:
Should we consider a null check in several of these symbol constructors?  To help catch this sort of mistake in the future?